### PR TITLE
Update DeadSpace.md

### PR DESCRIPTION
**Pull request by r3sus**
Described Free Camera Table by GhostInTheCamera

<details> <summary>P.S. Looking for original guide by Jim for ds3 (1) / table for ds2 (2).</summary>

(1) is referenced at [DET](https://web.archive.org/web/20140527235516/http://www.deadendthrills.com:80/forum/discussion/93/dead-space-3) and [NeoGAF](https://www.neogaf.com/threads/the-neogaf-screenshotters-compendium-thread.546220/page-7#post-86365225).
The [link to guide](http://www.flickr.com/groups/deadendthrills/discuss/72157631765632995/page2) showed to be [saved once](https://web.archive.org/web/2017*/http://www.flickr.com/groups/deadendthrills/discuss/72157631765632995/page2), however [not archived](https://web.archive.org/web/20171224231056/http://www.flickr.com/groups/deadendthrills/discuss/72157631765632995/page2/).
Please, advice, if there's any other way to try get this page or someone has it/guide locally. 

(2) wasn't shared publicly, altho it exists according to the [reply](https://web.archive.org/web/20160830145827/http://deadendthrills.com/forum/discussion/397/dead-space-2) and [screenshots](https://steamcommunity.com/id/neoenigma/screenshots/?appid=47780).

</details>

### CODE REVIEW

**Comment by originalnicodr** [comment]
Have you checked the DS3 table in the [cheat table archive](https://framedsc.github.io/cheattablearchive.htm#dead-space-3)?

And for DS2 I dont know if someone has jims original table, but ghostinthecamera has some [notes on it](https://github.com/ghostinthecamera/PhotomodeCheatTables/blob/master/WIP/Dead%20Space%202/notes.txt).

**Comment by r3sus** [comment]
Sure, I checked and knew it's here. But I was looking for Guide to it. :) Saw notes as well, but looking for ready tables first. And thanks for your reply.

File: markdown/GameGuides/DeadSpace.md [hunk 1]
```diff
@@ -12,15 +12,42 @@ Reshade | Yes
...
 DirectX versions | 9
 
-## Tools
+Resolution can be changed in configuration file.
+GeDoSaTo is supported.
+[(guide)](https://steamcommunity.com/sharedfiles/filedetails/?id=604010024)  
+
+## Free Camera
+[Table by GhostInTheCamera](https://github.com/ghostinthecamera/PhotomodeCheatTables/blob/master/WIP/Dead%20Space/GITC%20-%20Dead%20Space%20WIP.CT)  #
+[(download)](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/ghostinthecamera/PhotomodeCheatTables/blob/master/WIP/Dead%20Space/GITC%20-%20Dead%20Space%20WIP.CT)
```
Comment (FransBouma) [inline markdown/GameGuides/DeadSpace.md:21]: Please correct the url, it now goes via minhaskamal.github.io...

Comment (FransBouma) [inline markdown/GameGuides/DeadSpace.md:21]: Please remove the download link, as it pipes through an unknown script.

File: markdown/GameGuides/DeadSpace.md [hunk 2]
```diff
@@ -12,15 +12,42 @@ Reshade | Yes
...
+Turn Free Camera on/off with `Insert` and `Delete`:
+* for camera navigation/rotation - use `Num 1-9` and arrows.
+* change FOV with `Num +/-`.
+* char can be moved in that mode. 
+### Other features:  
+* Timeflow: turn  off/on with `Num 0/.` - used to pause (freeze) the game. 
+* Modify Timescale and Aspect Ratio.    
+
+### Hotkeys Modification
+[Cheat Table](https://gist.github.com/r3sus/93edc2c58a25841c30b6effcf7936257)  #
```
Comment (FransBouma) [inline markdown/GameGuides/DeadSpace.md:33]: Please include tables in the PR so they're in the repository. Don't link to gists which can be deleted at any moment.

File: markdown/GameGuides/DeadSpace.md [hunk 3]
```diff
@@ -12,15 +12,42 @@ Reshade | Yes
...
-* [Original Cheat Table by Jim2Point0](../CheatTables/Archive/deadspace.CT)
-* [Alternative table by GhostInTheCamera](https://github.com/ghostinthecamera/PhotomodeCheatTables/blob/master/WIP/Dead%20Space/GITC%20-%20Dead%20Space%20WIP.CT). 
+@alert tip  
+Add key delay in CE settings to prevent double press.  
+Customize hotkeys and their function by pressing `Ctrl+H` on selected entry.  
+@end
+
+## Fixed Camera Control
+* [Original Cheat Table by Jim2Point0](../CheatTables/Archive/deadspace.CT) - Camera Control.
+* [Modified Cheat Table](../CheatTables/deadspace_AoB.CT) - FOV, Camera, Char Size. Supports every game version by AoB scan.
```
Comment (FransBouma) [inline markdown/GameGuides/DeadSpace.md:46]: No need to keep 2 tables, 1 is enough.

**Comment by r3sus** [comment]
Ready, please review. 
Thanks.
P.S. looking for way to make hotkeys work only if game window is active, also no gravity or Zero-G mod.

**Pull request merged**